generator/pom.xml

```diff
@@ -146,15 +149,41 @@
             <artifactId>junit-jupiter-engine</artifactId>
         </dependency>
     </dependencies>
+
     <build>
         <pluginManagement>
             <plugins>
+
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-surefire-plugin</artifactId>
                     <version>3.0.0-M1</version>
                 </plugin>
+
+                <plugin>
+                    <artifactId>maven-assembly-plugin</artifactId>
+                    <executions>
+                        <execution>
+                            <phase>package</phase>
+                            <goals>
+                                <goal>single</goal>
+                            </goals>
+                        </execution>
+                    </executions>
+                    <configuration>
+                        <archive>
+                            <manifest>
+                                <mainClass>com.scottlogic.deg.generator.App</mainClass>
+                            </manifest>
+                        </archive>
+                        <descriptorRefs>
+                            <descriptorRef>jar-with-dependencies</descriptorRef>
```

Comment:
If this affects the filename can we call it 'generator'

Reply:
the final filename will be `generator-<VERSION>-jar-with-dependencies.jar`. 
it is created after the original `generator-<VERSION>.jar` has been created and does not replace it.

Reply:
sorry. understood now.
final jar file name is now `generator.jar`